lightning/src/events/mod.rs

```diff
@@ -1673,10 +1673,10 @@ impl Writeable for Event {
 				13u8.write(writer)?;
 				write_tlv_fields!(writer, {
 					(0, payment_id, required),
+					(1, path.trampoline_hops, optional_vec),
 					(2, payment_hash, option),
 					(4, path.hops, required_vec),
 					(6, path.blinded_tail, option),
-					(8, path.trampoline_hops, optional_vec),
```

Comment:
I think I steered you wrong on this one, sorry. Just noticed that blinded tails are even TLV so it would probably be more consistent to make the trampoline hops even as well. Can delete this fixup IMO.